docs/architecture/ADR-010-market-params.md

```diff
@@ -0,0 +1,163 @@
+# ADR 010: Market Parameters
```

Comment:
Is this going to specify a task or be a documentation? If latter, then we should organize it as we discussed recently. If former, then we should not re-copy definitions, and only include here what's new. Also, probably it should be formulated as the design doc.

Reply:
This is a design doc - it outlines decision + motivation + alternatives.